packages/turf-standard-deviational-ellipse/index.d.ts

```diff
@@ -0,0 +1,30 @@
+import { FeatureCollection, Feature, Coord, Polygon, Properties, Point } from '@turf/helpers';
+
+/**
+ * http://turfjs.org/docs/#standarddeviational-ellipse
+ */
+
+export interface SDEProps {
+    meanCenterCoordinates: Coord,
```

Comment:
Almost 😄 `Coord` is short for all 3 Point types, `Geometry<Point>`| `Feature<Point>` | `Position`.

What you're looking for is `Position` which is an `Array<number>` or more specific `[number, number]`.